docs/package.json

```diff
@@ -80,7 +80,7 @@
     "http-server": "^0.12.3",
     "jest": "26.1.0",
     "prettier": "^2.3.1",
-    "puppeteer": "^3.3.0",
+    "puppeteer": "^10.2.0",
```

Comment:
Wow ❗❗
Massive version upgrade 🚀 👏

Reply:
Heh, we only using few core methods, so the migration wasn't that hard as it might look. 😄